cas-server-core-authentication/src/main/java/org/apereo/cas/web/flow/AuthenticationExceptionHandler.java

```diff
@@ -72,38 +73,49 @@
         DEFAULT_ERROR_LIST.add(UnsatisfiedAuthenticationPolicyException.class);
     }
 
-    /** Ordered list of error classes that this class knows how to handle. */
-    
+    /**
+     * Ordered list of error classes that this class knows how to handle.
+     */
+
     private List<Class<? extends Exception>> errors = DEFAULT_ERROR_LIST;
 
-    /** String appended to exception class name to create a message bundle key for that particular error. */
+    /**
+     * String appended to exception class name to create a message bundle key for that particular error.
+     */
     private String messageBundlePrefix = DEFAULT_MESSAGE_BUNDLE_PREFIX;
 
     /**
      * Sets the list of custom exceptions that this class knows how to handle.
-     *
+     * <p>
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [<p> tag should be placed immediately before the first word, with no space after.](https://www.codacy.com/app/mmoayyed/cas/file/2770326140/issues/source?bid=3256115&fileBranchId=3357337#l89)